tests/e2e-permissions.spec.ts

```diff
@@ -0,0 +1,210 @@
+import { test, expect } from "./fixtures/lxd-test";
+import { visitInstance } from "./helpers/instances";
+import { gotoURL } from "./helpers/navigate";
+import { execSync } from "child_process";
+
+test.beforeEach(() => {
+  test.skip(
+    Boolean(process.env.CI),
+    "This suite is only run manually to create screenshots for the documentation",
+  );
+});
+
+test.describe("Given a user with Viewer Server permissions...", () => {
+  const instanceName = "test-instance";
+  const profileName = "test-profile";
+  const networkName = "test-network";
+  const poolName = "test-pool";
+  const volumeName = "test-volume";
+  //   const customISOName = "test-custom-iso"; // Need ISO file
+  //   const bucketName = "test-bucket"; // Need CephObject
+  const aclName = "test-acl";
+  const identityName = "test-identity";
+  const groupName = "test-group";
```

Comment:
We should use the randomized names, to allow retries on failures. I think we have helpers for all of these entities to generate the random names.